10_install_server/00_install_intro.txt

```diff
@@ -28,4 +28,3 @@ Slower growth over time.
 Reports logs can easily grow to 1.5GB per day.
 
 ======
-
```

Comment:
If I recall correctly, this blank line is necessary for the *old style* documentation to build correctly...